src/smpdflib/corrutils.py

```diff
@@ -64,10 +64,12 @@ def observable_correlations(results_table, base_pdf=None):
         M = np.concatenate([result._all_vals.as_matrix() for result in results])
         #Without atleast_2d would return a scalar if M has only one element.
         corrmat = np.atleast_2d(np.corrcoef(M))
-        title = "Observables correlation\n%s" % pdf
+        title = "Observables correlation\n%s" % pdf.label
```

Comment:
Having this plotting logic in here is kind of ugly, but we are not changing it now.